composer.json

```diff
@@ -16,7 +16,8 @@
         "php": ">=7.4",
         "symfony/framework-bundle": "~4.4 || ~5.0 || ~6.0",
         "symfony/monolog-bundle": "~3.0",
-        "symfony/expression-language": "~4.4 || ~5.0 || ~6.0"
+        "symfony/expression-language": "~4.4 || ~5.0 || ~6.0",
+        "monolog/monolog": "^2.4"
```

Comment:
Why not older monolog versions too ? [Symfony/monolog-bundle currently requires](https://github.com/symfony/monolog-bundle/blob/a41bbcdc1105603b6d73a7d9a43a3788f8e0fb7d/composer.json#L24)
```
"monolog/monolog": "^1.22 || ^2.0"
```

It should be fine, the current processor uses `array` type hints

Reply:
Because we have same on #12 => https://github.com/BedrockStreaming/MonologExtraBundle/pull/12/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R20
But I can update as you purpose

Reply:
done

Reply:
Because #12 uses `LogRecord` (available from `^2.4`) but not the current code on master ;)